.github/pull_request_template.md

```
@@ -1,48 +1,43 @@
-## Intent
+## Summary
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)